lessons/beginners/first-steps/index.md

```diff
@@ -55,10 +30,77 @@ NameError: name 'whoami' is not defined
 ```
 
 Tohle je *chybová hláška*, která se objeví vždycky,
-když uděláš něco špatně.
+když Python nebude spokojený.
 V průběhu kurzu jich uvidíš ještě spoustu,
 takže si ji dobře prohlédni, ať ji příště poznáš.
 
+## První příkaz
+
+Třemi „zobáčky“ `>>>` Python prosí o instrukce.
+Pojď mu nějakou dát!
+
+Ze začátku použij Pythonu jako kalkulačku.
+Za tři zobáčky napiš třeba `2 + 3` a zmáčkni <kbd>Enter</kbd>.
+
+``` pycon
+>>> 2 + 3
+5
+```
+
+Zobrazila se ti správná odpověď?
+Pokud ano, gratuluji! První příkaz v Pythonu máš za sebou.
+
+Zkusíš i odečítání?
+
+A jak je to s násobením?
+{# XXX: Jak zapsat násobení? `4 x 5` `4 . 5` `4 × 5` `4 * 5` -#}
```

Comment:
proč je tady tenhle komentář? pokud to je todo pro příští roky, obávám se, že ho nepochopíme. Navíc pod tím je věta, která ho částečně řeší.

Reply:
Máš pravdu, komentář je z dřívější verze.